stable/yugabyte/templates/service.yaml

```diff
@@ -426,11 +426,37 @@ spec:
         {{- if $root.Values.tls.enabled }}
         - name: {{ $root.Values.oldNamingStyle | ternary (printf "%s-yugabyte-tls-cert" .label) (printf "%s-%s-tls-cert" (include "yugabyte.fullname" $root) .label) }}
           secret:
+            {{- if not $root.Values.tls.certManager.enabled }}
             secretName: {{ $root.Values.oldNamingStyle | ternary (printf "%s-yugabyte-tls-cert" .label) (printf "%s-%s-tls-cert" (include "yugabyte.fullname" $root) .label) }}
+            {{- else }}
+            secretName: yugabyte-{{ .name }}-tls
```

Comment:
After modifying the cert secretName to match the existing secretName (to support new naming convention), we'll have to rework this logic slightly.